lib/validators/spec.js

```diff
@@ -1,348 +1,16 @@
-const util = require('../util');
```

Comment:
Lots of noise in this file but everything that was here now exists in `lib/validators/spec/swagger.js`.